### ISSUE TITLE
Allow file: with @libsql/client/web

### ISSUE THREAD

**Issue by arjunyel** [libsql/libsql-client-ts#21]
When locally developing with Cloudflare Workers it would be cool if I could use a local sqlite instance with `file:`

Here is how local sqlite is implemented in Miniflare https://github.com/cloudflare/miniflare/blob/master/packages/shared/src/sqlite.ts

**Comment by honzasp** [comment]
I'm not sure what's the proper solution here, because we have some contradictory requirements:
- The `@libsql/client/web` import is explicitly intended to use only the Web-standard APIs (`fetch` and `WebSocket`)
- The normal `@libsql/client` import will not work on Workers, because it uses Node bindings to SQLite
- Miniflare aims to emulate Workers as closely as possible, so we need to use hacks to escape the simulation and use Node bindings to SQLite.
- These hacks will likely not work when Miniflare [uses the `workerd` runtime and not Node](https://blog.cloudflare.com/miniflare-and-workerd/).

In other words, I understand that using a SQLite database for local development of Workers would be quite convenient, but it goes against the point of Miniflare, which aims to locally provide the same environment as Workers. I'm not sure how to resolve this; maybe we could provide a Worker-specific import such as `@libsql/client/worker`, which would support the Miniflare-specific hacks allowing the use of `file:` URLs locally?
